testing.go

```diff
@@ -93,6 +93,10 @@ func (test *Test) End() {
 	if test.t.Failed() {
 		test.span.SetTag("test.status", "FAIL")
 		test.span.SetTag("error", true)
+		test.span.LogFields(
+			log.String(EventType, EventTestFailure),
+			log.String(EventMessage, "Test has failed"),
```

Comment:
We don’t have the failure message?

Reply:
I guess not.

I have only the boolean:
![image](https://user-images.githubusercontent.com/69803/64127214-03e8ba80-cdb1-11e9-9459-d7db2311a34a.png)

Maybe if we in the future intercept the `Fatal` and `Fatalf` we can use that message. But the `*testing.T` doesn't have that info

Reply:
Because the `Fail()` method doesn't have any argument for a reason.

Reply:
Ok, good enough for now